server/routes/index.js

```diff
@@ -4,4 +4,11 @@
 
 'use strict';
 
-module.exports = [].concat(require('./base'), require('./auth'), require('./search'), require('./visits'));
+// TODO just sniff out all the files under /routes?
```

Comment:
Yeah, I imagine that should work as long as we don't need recursion and only ever have *.js files in this directory.